back/app/controllers/web_api/v1/permissions_controller.rb

```diff
@@ -69,6 +69,17 @@ def custom_fields
     render json: WebApi::V1::CustomFieldSerializer.new(fields, params: jsonapi_serializer_params_with_locked_fields).serializable_hash.to_json
   end
 
+  def custom_field_options
```

Comment:
I ended up creating a new endpoint. The FE was expecting the `custom_fields` endpoint (see line 66 of this file) to include the options. But clearly it is not, and because `user_requirements_service.requirements_custom_fields @permission` returns an array of fields and not an `ActiveRecord` query, we cannot include them in the standard rails way.